css/settings.css

```diff
@@ -46,5 +47,27 @@ input{
 
 #appointments h3{
     font-weight: 600;
+    font-size: 18px;
     margin-bottom: unset;
+}
+
+.header{
+    display: flex;
+}
+
+.save{
+    margin-left: auto;
```

Comment:
If you set the margin-left to 20rem instead, at least for now, until we have the sidebar. On bigger screen it looks a bit off that the save button is way to the left.